feat(Comments): send webhooks

J9rem opened the pull request:
cette PR est liée à la PR du coeur https://github.com/YesWiki/yeswiki/pull/967 sur les commentaires

Elle permet d'attraper les événements liées aux commentaires (événements définis via EventDispatcher de la librairie symfony, cf. la PR du coeur)

et ensuite, elle permet aussi de formater en envoyer des webhooks associés aux commentaires

_PR draft en attendant la fusion de la P du coeur_

**Ce que ça fait**:
 - décoration du `CommentService` pour éviter les erreurs en cas d'installation de Webhooks avec cette PR sur des versions antérieures à `4.2`
 - mise en place du suivi des commentaires dans `WebhooksController`
 - création de template `twig` pour gérer le formatage des messages des webhooks
 - ajout de l'option pour suivre les commentaires au sein de l'interface présente en bas de l'action `bazar`

J9rem commented:
Je viens de faire un rebase.
Les présentes modifications peuvent être déployées dès maintenant même si le système de `EventDispatcher` n'est pas encore actif dans le coeur car j'utilise une `factory` qui choisit la bonne classe à utiliser en fonction de l'existence ou non d'un `EventDispatcher`

J9rem committed:
fix(webhooks_form): show 'comments' only if EventDispatcher exists

mrflos commented:
hello @J9rem , n'hésites pas à merger si c'est stable ou me prévenir quand il faut reviewer/merger si tu preferes

J9rem commented:
J'allais justement te prévenir.
Mais vu que c'est très stable, je vais me permettre de fusionner (et on demandera de faire des tests avec `doryphore-test` avant de faire un tag)

J9rem committed:
feat(Comments): send webhooks

+ refactor(WebhookController): implement interface only if exist
without usage of a factory
